wordpress-mobile/WordPress-Android
# Expand description in site address help image

Fixes #6464 

The site address help popup has an image to show the user where to find their site address. This PR expands the `contentDescription` of the image to provide a textual description of where to find the site address.

To test:
* On a physical device with Talkback enabled and app data cleaned, launch the app
* Tap "LOG IN"
* Tap "Log in to your site by entering your site address instead"
* Tap "Need help finding your site address?"
* Tap on the image on the popup and Talk back should start announcing "The site address can be found in the navigation bar of your web browser"
